bitnami/external-dns/values.yaml

```
@@ -480,9 +480,12 @@ transip:
   ## @param transip.account When using the TransIP provider, specify the account name.
   ##
   account: ""
-  ## @param transip.apiKey When using the TransIP provider, specify the API key to use.
+  ## @param transip.apiKey When using the TransIP provider, specify the API key to use. Mutually excluse with apiKeySecret.
   ##
   apiKey: ""
+  ## @param transip.apiKeySecret When using the TransIP provider, specify the secret containing the API key to use. Mutually exclusive with apiKey.
+  ##
+  apiKeySecret: ""
```

Follow the same format used for other providers:
```suggestion
  ## @param transip.secretName secretName Use an existing secret with API key to use defined.
  ##
  secretName: ""
```